source/MultiGrid/MultiGrid.js

```diff
@@ -433,7 +434,7 @@ export default class MultiGrid extends Component {
         columnCount={columnCount - fixedColumnCount}
         columnWidth={this._columnWidthRightGrid}
         height={this._getBottomGridHeight(props)}
-        onScroll={this._onScroll}
+        onScroll={(info) => {this._onScroll(info);onScroll && onScroll(info)}}
```

Comment:
Oh right, i thought the _onScroll has been used not only in bottomRight grid, also in other grid. And the passed onScroll only should use in one Grid.
I will update it.